hawkular-nest/hawkular-nest-feature-pack/pom.xml

```diff
@@ -96,6 +96,100 @@
   <build>
     <plugins>
 
+      <!-- Unpack the upstream artifacts that host standalone*.xml stubs that we want to adapt in the next steps -->
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-dependency-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>unpack-deps</id>
+            <phase>generate-resources</phase>
+            <goals>
+              <goal>unpack</goal>
+            </goals>
+            <configuration>
+              <artifactItems>
+                <artifactItem>
+                  <groupId>org.hawkular.commons</groupId>
+                  <artifactId>hawkular-bus-feature-pack</artifactId>
+                  <type>zip</type>
+                  <includes>configuration/standalone/template.xml,configuration/standalone/subsystems.xml</includes>
+                  <outputDirectory>${project.build.directory}/feature-pack-resources-tmp</outputDirectory>
+                </artifactItem>
+              </artifactItems>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+
+      <!-- Generate the standalone*.xml stubs from the upstream originals using XSL transformations -->
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>xml-maven-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>transform-xmls</id>
+            <phase>generate-resources</phase>
+            <goals>
+              <goal>transform</goal>
+            </goals>
+            <configuration>
+              <transformationSets>
+                <transformationSet>
+                  <dir>${project.build.directory}/feature-pack-resources-tmp</dir>
+                  <stylesheet>${basedir}/src/main/xsl/configuration/standalone/hawkular-nest-subsystems.xsl</stylesheet>
+                  <includes>
+                    <include>configuration/standalone/subsystems.xml</include>
+                  </includes>
+                  <outputDir>${project.build.directory}/feature-pack-resources</outputDir>
+                  <fileMappers>
+                    <fileMapper implementation="org.codehaus.plexus.components.io.filemappers.RegExpFileMapper">
+                      <pattern>^configuration/standalone/(.*).xml$</pattern>
```

Comment:
Also a good question. Nobody asked for other than plain `standalone` yet. Once the requirement comes, we know how to add those other files.